assemblies/assembly_openshift-immutable/main.adoc

```diff
@@ -11,31 +11,39 @@ include::_artifacts/author-group.adoc[]
 
 // Purpose statement for the assembly
 As a system engineer, you can deploy a {PRODUCT} immutable server environment on {OPENSHIFT}
-to provide an infrastructure to execute processes and other business assets. You can use
+to provide an infrastructure to execute {ASSETS} and other business assets. You can use
 standard integration tools to manage the immutable {KIE_SERVER} image. You can create new server images
```

Comment:
If the server is immutable, should it be managed by this integration tools you mentioned? (still not understanding what it means.)